bases/br_inep_censo_escolar/escola/publish.sql

```diff
@@ -0,0 +1,164 @@
+/*
+
+Query para publicar a tabela.
+
+Esse é o lugar para:
+    - modificar nomes, ordem e tipos de colunas
+    - dar join com outras tabelas
+    - criar colunas extras (e.g. logs, proporções, etc.)
+
+Qualquer coluna definida aqui deve também existir em `table_config.yaml`.
+
+# Além disso, sinta-se à vontade para alterar alguns nomes obscuros
+# para algo um pouco mais explícito.
+
+TIPOS:
+    - Para modificar tipos de colunas, basta substituir STRING por outro tipo válido.
+    - Exemplo: `SAFE_CAST(column_name AS NUMERIC) column_name`
+    - Mais detalhes: https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types
+
+*/
+
+CREATE VIEW basedosdados-dev.br_inep_censo_escolar.escola AS
+SELECT 
+SAFE_CAST(ano AS INT64) ano,
+SAFE_CAST(sigla_uf AS STRING) sigla_uf,
+SAFE_CAST(id_municipio AS INT64) id_municipio,
+SAFE_CAST(id_orgao_regional AS INT64) id_orgao_regional,
```

Comment:
O que é essa variável? Por que tem tanto missing?

Reply:
Muitas variaveis de identificação tem muito missing no começo dos anos.
Soluções : 1) talvez de pra mergir dentro da propria base para corrigir isso(?), supondo que isso nao muda ao longo do tempo. 2) ou ainda criar mudar o diretorio para adicionar essa variável. 3) excluir a variável pois não harmoniza com outras bases de escola

Reply:
Depois me confere se é so essa modificação e a questão das datas que aí eu libero a base completa na bigquery que vai me ajudar até criar espaço no computador!

Reply:
Vamos deixar então essa `id_orgao_regional` como está por enquanto. Mudamos no futuro se for o caso.

De minha parte são esses comentários só. Ainda falta publicar a versão atualizada no BQ né?

Reply:
isso! to terminando a base 3, a mais dificilzinha vai ser a 4 pelo tamanho mas em breve dou um jeito. Se forem só essas mudanças já vou mudar para as completas na BQ!